packages/@o3r/application/schematics/ng-add/index.ts

```diff
@@ -57,8 +57,11 @@ function ngAddFn(options: NgAddSchematicsSchema): Rule {
         return tree;
       }
 
+      const addImportToModuleFile = (name: string, file: string, moduleFunction?: string) => additionalRules.push(
```

Comment:
it's a bit confusing to have a function with the same name as the one from @o3r/schematics